code/game/machinery/spaceheater.dm

```diff
@@ -41,6 +41,10 @@
 	///Should we add an overlay for open spaceheaters
 	var/display_panel = TRUE
 
+	var/maxTemp = settable_temperature_median + settable_temperature_range - T0C
```

Comment:
don't put these here, just put them in the AltClick proc directly because they aren't used anywhere else, they don't need to be vars, just assign them directly to target_temperature